tests/std/tests/GH_000527_remove_allocator_void/test.compile.pass.cpp

```diff
@@ -48,19 +49,16 @@ STATIC_ASSERT(can_allocate<allocator<int>>);
 STATIC_ASSERT(is_convertible_v<allocator<void>, allocator<int>>);
 
 #if _HAS_CXX20
-STATIC_ASSERT(has_member_size_type<allocator<void>>);
-STATIC_ASSERT(has_member_difference_type<allocator<void>>);
-STATIC_ASSERT(has_member_pocma<allocator<void>>);
-STATIC_ASSERT(has_member_is_always_equal<allocator<void>>);
-STATIC_ASSERT(can_allocate<allocator<void>>);
-STATIC_ASSERT(is_convertible_v<allocator<int>, allocator<void>>);
+constexpr bool has_cxx20 = true;
 #else
-STATIC_ASSERT(!has_member_size_type<allocator<void>>);
-STATIC_ASSERT(!has_member_difference_type<allocator<void>>);
-STATIC_ASSERT(!has_member_pocma<allocator<void>>);
-STATIC_ASSERT(!has_member_is_always_equal<allocator<void>>);
-STATIC_ASSERT(!can_allocate<allocator<void>>);
-STATIC_ASSERT(!is_convertible_v<allocator<int>, allocator<void>>);
+constexpr bool has_cxx20 = false;
 #endif
 
+STATIC_ASSERT(has_cxx20 == has_member_size_type<allocator<void>>);

```

Comment:
Pure style, and totally not worth resetting testing if we change nothing else: I would have written these as `[Actual value] == [Expected value]` instead of the other way 'round.

Reply:
That's my preferred style too - but I did it this way so that they would line up, and make the pattern obvious.